discovery-infra/tests/base_test.py

```diff
@@ -258,3 +258,57 @@ def wait_for_nodes_status_installing_or_installed(
                       consts.NodesStatus.INSTALLED],
             interval=30,
         )
+
```

Comment:
I think base test starts to be too large file, lets create cluster object and move all cluster functionality to it